cmd/kubeadm/app/cmd/join.go

```diff
@@ -31,35 +31,36 @@ import (
 )
 
 var (
-	join_done_msgf = dedent.Dedent(`
+	joinDoneMsgf = dedent.Dedent(`
 		Node join complete:
 		* Certificate signing request sent to master and response
 		  received.
 		* Kubelet informed of new secure connection details.
 
-		Run 'kubectl get nodes' on the master to see this node join.
+		Run 'kubectl get nodes' on the master to see this machine join.
```

Comment:
well, the machine "becomes" a node when it joins... so maybe this was OK before? Also the `kubectl get nodes` command refers to nodes...